packages/core/src/utils/TypeUtil.ts

```diff
@@ -104,6 +104,7 @@ export type SdkVersion =
   | `${'html' | 'react' | 'vue'}-ethers5-${string}`
   | `${'html' | 'react' | 'vue'}-ethers-${string}`
   | `${'html' | 'react' | 'vue'}-solana-${string}`
+  | `${'html' | 'react' | 'vue'}-multichain-${string}`
```

Comment:
Review this later, right?

Reply:
Totally, will be handled in next PR. Here is also tech doc about it: https://www.notion.so/walletconnect/Multi-chain-SDK-parameters-refactor-for-Blockchain-API-endpoints-6b200e4f05e2465aa9e6785faef9ecdc?pvs=4